test/replication-luatest/gh_10561_anon_replicas_dont_participate_in_elections_test.lua

```diff
@@ -0,0 +1,82 @@
+local t = require('luatest')
```

Comment:
Nit: we can consider moving the test to `anon_test`. Up to you

Reply:
Moved.